src/xCacheLibrary/XCache.cs

```diff
@@ -323,7 +323,7 @@ public bool TryGetValue(TKey key, out TValue value)
     /// <summary>
     /// Starts the background cleanup task to expire the cache at fixed intervals.
     /// </summary>
-    private async void StartCleanUpExpiredEntriesTask()
```

Comment:
It doesn't return a Task, so test runners can't wait for it to complete. If an exception is thrown inside it, it crashes the process instead of being handled.